core/scripts/chaincli/handler/handler.go

```diff
@@ -130,14 +156,304 @@ func (k *Keeper) sendEth(ctx context.Context, to common.Address, amount *big.Int
+	}
+
+	var found bool
+	for _, ntwrk := range existingNetworks {
+		if ntwrk.Name == networkName {
+			found = true
+			break
+		}
+	}
+
+	if !found {
+		if _, err = dockerClient.NetworkCreate(ctx, networkName, types.NetworkCreate{}); err != nil {
+			return "", nil, fmt.Errorf("failed to create network: %s", err)
+		}
+	}
+
+	// Create DB container
+	postgresContainerName := fmt.Sprintf("%s-postgres", containerName)
+	dbContainerResp, err := dockerClient.ContainerCreate(ctx, &container.Config{
+		Image: defaultPOSTGRESImage,
+		Cmd:   []string{"postgres", "-c", `max_connections=1000`},
+		Env: []string{
+			"POSTGRES_USER=postgres",
+			"POSTGRES_PASSWORD=development_password",
+		},
+		ExposedPorts: nat.PortSet{"5432": struct{}{}},
+	}, nil, &network.NetworkingConfig{
+		EndpointsConfig: map[string]*network.EndpointSettings{
+			networkName: {Aliases: []string{postgresContainerName}},
+		},
+	}, nil, postgresContainerName)
+	if err != nil {
+		return "", nil, fmt.Errorf("failed to create Postgres container: %s", err)
+	}
+
+	// Start container
+	if err = dockerClient.ContainerStart(ctx, dbContainerResp.ID, types.ContainerStartOptions{}); err != nil {
+		return "", nil, fmt.Errorf("failed to start DB container: %s", err)
+	}
+	log.Println("Postgres docker container successfully created and started: ", dbContainerResp.ID)
+
+	time.Sleep(time.Second * 10)
+
+	// Pull node image if needed
+	if _, _, err = dockerClient.ImageInspectWithRaw(ctx, defaultChainlinkNodeImage); err != nil {
+		log.Println("Pulling node docker image...")
+		if out, err = dockerClient.ImagePull(ctx, defaultChainlinkNodeImage, types.ImagePullOptions{}); err != nil {
+			return "", nil, fmt.Errorf("failed to pull node image: %s", err)
+		}
+		out.Close()
+		log.Println("Node docker image successfully pulled!")
+	}
+
+	// Create temporary file with chainlink node login creds
+	apiFile, passwordFile, fileCleanup, err := createCredsFiles()
+	if err != nil {
+		return "", nil, fmt.Errorf("failed to create creds files: %s", err)
+	}
+
+	// Create container with mounted files
+	portStr := fmt.Sprintf("%d", port)
+	nodeContainerResp, err := dockerClient.ContainerCreate(ctx, &container.Config{
+		Image: defaultChainlinkNodeImage,
+		Cmd:   []string{"local", "n", "-p", "/run/secrets/chainlink-node-password", "-a", "/run/secrets/chainlink-node-api"},
+		Env: append([]string{
+			"DATABASE_URL=postgresql://postgres:development_password@" + postgresContainerName + ":5432/postgres?sslmode=disable",
+			"ETH_URL=" + h.cfg.NodeURL,
+			fmt.Sprintf("ETH_CHAIN_ID=%d", h.cfg.ChainID),
+			"LINK_CONTRACT_ADDRESS=" + h.cfg.LinkTokenAddr,
+			"DATABASE_BACKUP_MODE=lite",
+			"SKIP_DATABASE_PASSWORD_COMPLEXITY_CHECK=true",
+			"LOG_LEVEL=debug",
+			"MIN_OUTGOING_CONFIRMATIONS=2",
+			"CHAINLINK_TLS_PORT=0",
+			"SECURE_COOKIES=false",
+			"GAS_ESTIMATOR_MODE=BlockHistory",
+			"ALLOW_ORIGINS=*",
+			"DATABASE_TIMEOUT=0",
+			"KEEPER_CHECK_UPKEEP_GAS_PRICE_FEATURE_ENABLED=true",
```

Comment:
i don't think we need this. this was an experimental feature, not used

Reply:
removed